app/models/pieces/bishop.rb

```diff
@@ -3,5 +3,6 @@ class Bishop < Piece
   
   def set_defaults
     self.icon = (self.color == 1) ? "&#9821;" : "&#9815;"
+    self.first_move ||= true
```

Comment:
what does first_move indicate? do you mean sth like 'first_move_made' which is true or false.. I would suggest to rename it to 'moved' which is in my understanding more clear.. but open for discussion :)